client/src/components/admin/AdminDashboard.tsx

```
@@ -0,0 +1,123 @@
+          {activeSection === 'verification' && 'Verification Requests'}
+          {activeSection === 'reports' && 'Reported Profiles'}
+          {activeSection === 'support' && 'Support Requests'}
+        </h1>
+      </div>
+
+      {/* Overlay for mobile sidebar */}
+      {sidebarOpen && (
+        <div
+          onClick={() => setSidebarOpen(false)}
+          className="fixed inset-0 bg-black bg-opacity-50 z-10 md:hidden"
+        ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)